pkg/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -9,10 +9,20 @@ package v1alpha1
 import (
 	"net/url"
 
+	"github.com/talos-systems/talos/cmd/osctl/pkg/client/config"
 	"github.com/talos-systems/talos/pkg/config/machine"
 	"github.com/talos-systems/talos/pkg/crypto/x509"
 )
 
+// ConfigBundle defines the group of v1alpha1 config files.
+//docgen: nodoc
+type ConfigBundle struct {
+	InitCfg         *Config
+	ControlPlaneCfg *Config
+	JoinCfg         *Config
+	TalosCfg        *config.Config
+}
+
```

Comment:
super nit: should this be in `v1alpha1_configurator_bundle.go`?

Reply:
moved in amended commit.